docker/ubuntu/kocervantes/build_cervantes_tc.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Building toolchains for cervantes..."
+git clone https://github.com/koreader/koxtoolchain.git
+pushd koxtoolchain
+git checkout b212bd2f61fcbe0a2bc2d3c4085d7b95a4f79a48
+
+./gen-tc.sh cervantes
```

Comment:
This changes are needed to build a toolchain under docker. I checked that. I have no idea of further steps :p

Reply:
Next you or I (or whoever) push it to Docker cloud so it can be used in CI. After I look at it properly on laptop or desktop rather than phone anyway. ;-)